### ISSUE TITLE
Cookbook fails with chef 13 because of redisio

### ISSUE THREAD

**Issue by MaximKraev** [sensu/sensu-chef#583]
cookbook files with error
```
[2017-10-18T17:29:47+00:00] ERROR: redisio_install[redis-installation] (redisio::install line 32) had an error: NameError: uninitialized constant Chef::Resource::RedisioInstall
[2017-10-18T17:29:47+00:00] FATAL: Chef::Exceptions::ChildConvergeError: Chef run process exited unsuccessfully (exit code 1)
```

## Possible Solution
looks `redisio` is dead and should be replaced
https://github.com/brianbianco/redisio/issues/319

## Your Environment

* Version of this cookbook used: 4.0.6
* Version of Chef used: 13.4.24
* Operating System and version (e.g. CentOS 7, Ubuntu 14.04): Ubuntu 14.04

**Comment by tmonk42** [comment]
The fix is in master for the redisio cookbook as of Aug 15th, still needs a release and publish to supermarket: https://github.com/brianbianco/redisio/pull/350

**Comment by majormoses** [comment]
I have become a maintainer for redisio cookbook to unblocked us. I have released version 2.7.0 which has now been pulled in and released in:
- https://github.com/sensu/sensu-chef/releases/tag/v4.3.0
- https://supermarket.chef.io/cookbooks/sensu/versions/4.3.0

Let me know if you run into any issues.
